package-lock.json

```diff
@@ -8236,7 +8236,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
looks like you've got some unrelated package-lock changes here, make sure they dont get merged in